Mathlib/CategoryTheory/Abelian/Transfer.lean

```diff
@@ -198,4 +204,148 @@ def abelianOfEquivalence {C : Type u₁} [Category.{v} C] [Preadditive C] [HasFi
   abelianOfAdjunction F F.inv F.asEquivalence.unitIso.symm F.asEquivalence.symm.toAdjunction
 #align category_theory.abelian_of_equivalence CategoryTheory.abelianOfEquivalence
 
+namespace transfer_enough_injectives
+
+variable {𝒜: Type u₁} {ℬ : Type u₂} [Category.{v₁} 𝒜] [Category.{v₂} ℬ]
+variable [EnoughInjectives ℬ]
```

Comment:
Do the constructions here really all need `EnoughInjectives`, or does it suffice to mostly work with an injective hull for some fixed object, until the very end where you prove that you can transfer `EnoughInjectives` along the adjunction?

Reply:
Indeed, assuming just an injective hull is sufficient